REQUIRE

```diff
@@ -1,7 +1,12 @@
-julia 0.6
-JuMP 0.18
+julia
+Nullables
+SpecialFunctions
+LinearAlgebra
```

Comment:
The package should switch to Project.toml

Reply:
I've added the Project.toml file